### ISSUE TITLE
Voiceover reading button for "Click here to View full recipe" link.

### ISSUE THREAD

**Issue by Karan4495** [dotnet/maui#5816]
GitHubTags:
#.NET MAUI; #A11ySev3; #A11yMAS; #A11yTCS; #Benchmark; #iOS; #iPad; #WCAG4.1.2; #GH_.NETMAUI_IOS_Mar2022; #Name Role Value​; #AINotInScope;

### Description

Voiceover reading incorrect role for "Click here to View full recipe" i.e. Voiceover reading button for "Click here to View full recipe" link.

### Steps to Reproduce

1.Install and open ".NET MAUI" app.
2.Swipe till "search box".
3.Enter Recipe name in search box and search.
4.Swipe till any recipe and double tap.
5."Recipe" Screen will open.
6.Verify Voiceover reading Correct role for "Click here to View full recipe" link or Not.

https://user-images.githubusercontent.com/100186395/161683803-46b93570-445e-41b9-b460-40ff6ca8f5ba.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15.4(iPad)

### Relevant log output

```shell
Voiceover should read correct role for "Click here to View full recipe" i.e. Voiceover read Link for "Click here to View full recipe" link.
```

**Comment by Karan4495** [comment]
Verified issue in ".NET MAUI" app. issue is Not fixed.
IOS Version: iPad(15.5)
As its Sev3 bug it is not required to fix for compliance, and has no SLA. Product team can keep this bug to their back log.
added attachment for references.

https://user-images.githubusercontent.com/100186395/179227642-c37e99fa-213b-45db-a431-10d2be003f49.mp4

**Comment by rachelkang** [comment]
Hi, @Karan4495 - the "Click here to view full recipe" was a link at first, but now it is a button. Since it is a button, it makes sense to me that Voiceover also announces it as a button. Could you please let me know if I'm misunderstanding something, or if this can be closed? Thank you

**Comment by Karan4495** [comment]
Hi, @rachelkang Its not related to UI of the "Click here to view full recipe" Link when we double tap on "Click here to view full recipe" the recipe get open in web/Safari so the role should be define as link.

**Comment by merriemcgaw** [comment]
@PureWeen / @Redth / @rachelkang  - is this issue in the sample app, or are we looking at not being able to fix it before we ship GA as a platform issue?

**Comment by rachelkang** [comment]
@merriemcgaw @Karan4495 We've observed mixed approaches to this scenario on iOS platform-specific apps, including the exact scenario we had here.  Regardless, we've updated the Recipes app to use the suggested approach and announce as a "link"!

**Comment by merriemcgaw** [comment]
@Karan4495 can you please verify so that @rachelkang can close the issue?

**Comment by Karan4495** [comment]
Verified issue in ".NET MAUI" app. issue fixed so closing the bug.
IOS Version: iPad(15.6.1)

https://user-images.githubusercontent.com/100186395/188826245-4b36e3b7-de4f-4e32-890b-b27c79a0ab3d.MP4
